.github/workflows/on-pr.yml

```diff
@@ -23,7 +23,6 @@ concurrency:
 jobs:
   info:
     name: info
-    if: ${{ github.event.pull_request.head.repo.full_name == github.repository }}
```

Comment:
Is this enabling these steps for community PRs?

Reply:
D'oh, I removed the wrong thing here.